docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md

```diff
@@ -2,54 +2,74 @@
 
-If there is a feature or change that you would like to see in Web3 Onboard, you can fork the repo and make a pull request to have the changes merged in to the main repo and released as part of the official packages.
+## [Pull Requests 🗂️](#pr)
 
-Once a PR is created, one of the Web3 Onboard maintainers will acknowledge the PR and add it to our sprint planning to be reviewed as soon as possible.
+To create a new feature or change in Web3 Onboard, fork the repo and make a pull request on the [develop branch](https://github.com/blocknative/web3-onboard/tree/develop) to have your changes merged in and released as part of the official packages.
 
+Once a PR is created, a Web3 Onboard maintainer will acknowledge it and add to sprint planning for review as soon as possible.
 
-### Package Versioning
+### Package Versioning 📦
 
-After making changes within a web3-onboard package you will want to bump the version of the specific package. For example if you were to add a new injected wallet to the injected package you will need to bump the version of the injected package within that module's [package.json](https://github.com/blocknative/web3-onboard/blob/8531a73d69365f7d584320f1c4b97a5d90f1c34e/packages/injected/package.json#L3). 
+We use both semantic and `alpha` versioning for publishing and testing packages prior to official release.  If you do not see an `-alpha.x` tag on the version of the package you made changes in, you will need to bump the version AND add an alpha flag followed by an alpha version number starting at 1 (ex. `-alpha.1`).
 
-We use both semantic and `alpha` versioning for publishing and testing packages prior to official release.  This means if you do not see an `-alpha.x` tag on the version of the package you made changes in you will need to not only bump the version but also add an alpha flag followed by an alpha version number starting at 1 (ex. `-alpha.1`).
+After making changes within a web3-onboard package, bump the version of the specific package:
 
 Example : Adding a new injected wallet to the `injected` package -
 [Injected Package.json](https://github.com/blocknative/web3-onboard/blob/develop/packages/injected/package.json#L3) changes : `"version": "2.2.4",` --> `"version": "2.3.0-alpha.1",`
 
 Example 2 : Bug fix within the `core` package -
 [Core package.json](https://github.com/blocknative/web3-onboard/blob/8531a73d69365f7d584320f1c4b97a5d90f1c34e/packages/core/package.json#L3) changes: `"version": "2.9.1-alpha.1",` --> `"version": "2.9.1-alpha.2",`
 
+## [How can I add a new wallet? 💳](#new-wallet)
 
-### How can I get a new wallet added?
+To add a new wallet to the official Web3 Onboard packages and repo, create a [pull request](#pull-requests-🗂️). Web3 Onboard does not require a wallet to be a part of the main code, so a separate wallet module can be created without any changes to the Web3Onboard codebase. Your pull request must include a README for the package.  
```

Comment:
Might be good to add something like`quality README` and something around this being the first point of contact for dapp devs...wallet devs seem to put the smallest effort into documentation lol

Reply:
haha noted. I'll add some emphasis.